config/kubernetes/sig-contributor-experience/teams.yaml

```diff
@@ -47,6 +47,18 @@ teams:
     - Phillels
     - tpepper
     privacy: closed
+  project-board-maintainers:
```

Comment:
Out of curiosity, is this team being used anywhere?

Reply:
Never mind, figured it out: https://github.com/orgs/kubernetes/teams/project-board-maintainers/projects :)